### ISSUE TITLE
new metric: Mean Directional Accuracy

### ISSUE THREAD

**Issue by ericleonardo** [oremnirv/Taylorformer#1]
Hi... Please, is it possible for you to include new metric Mean Directional Accuracy?
Ex: in the Exchange dataset, predicting the Direction (Up/Down) of price is much more important than MSE.
Approach:
if (True_0 - True_1) * (Predicted_0 - Predicted_1) > 0, return 1 (correct diff signal prediction: both risen, or both fallen: positive multiply)
if (True_0 - True_1) * (Predicted_0 - Predicted_1) < 0, return 0 (wrong diff signal prediction: one risen and other fallen: negative multiply)
return mean of all predictions. 

So it could include the MDA metric to see if Taylorformer can predict more than 50% MDA (50% is random guessing up/down).
I already tried to code the MDA metric for Tensorflow, Keras, Torch, but it is much more easier for SK Learn.

Question from forum:
[https://stackoverflow.com/questions/76885568/custom-metrics-in-keras-to-evaluate-directional-accuracy](https://stackoverflow.com/questions/76885568/custom-metrics-in-keras-to-evaluate-directional-accuracy)
[https://stackoverflow.com/questions/74671602/tensorflow-mda-custom-loss-and-valueerror-no-gradients-provided-for-any-variabl](https://stackoverflow.com/questions/74671602/tensorflow-mda-custom-loss-and-valueerror-no-gradients-provided-for-any-variabl)

Thank you and Congratulations for the great work!
